### ISSUE TITLE
Feat: Add support for SIDBlaster / HardSID

### ISSUE THREAD

**Issue by ralph303** [Alexco500/sidplay5#8]
Is it possible to add support for SIDBlaster (via HardSID lib) in Sidplay?
The "old" Sidplay code base supported HardSID in versions lower than 4, but the latest of the old versions 3.4.1 does not run on Apple Silicon.

If this is not possible, maybe someone who reads this can recommend a good player for MacOS. I only know of JSIDPlay2 for MacOS so far. Thank you.

**Comment by Alexco500** [comment]
Sounds interesting. I know about JSIDPlay2 (from Forum64).
I guess to support that reliable, I would at least have access to one unit (or bombard someone with debug-releases until it works :-) )

**Comment by ralph303** [comment]
Feel free to write me on Forum64 (hiTCH-HiKER) and I can get a SIDBlaster-USB-device for you. It may take two or three weeks, because they are currently unavailable but I think its the best solution to have your own one for testing purposes.

**Comment by bluecursor** [comment]
I also would like this feature added (SIDBlaster-USB)

**Comment by gr8bit** [comment]
@ralph303 @Alexco500 did you guys make that happen yet? I ordered 2 units myself lately ;)

**Comment by Alexco500** [comment]
Yes, it should work, but songs with digi samples slow down a lot. I still need to figure out why.

**Comment by gr8bit** [comment]
It works, I'm just having power supply issues with the SIDBlaster though (starts up every tenth time), but that's none of SIDplay's issues or fault. ;)
Are the digs slowdowns maybe related to the hardware not allowing for the many updates samples need to $D418?
